packages/amplify-console-integration-tests/src/consoleHosting/consoleHosting.ts

```diff
@@ -1,6 +1,89 @@
 import { nspawn as spawn } from 'amplify-e2e-core';
-import * as nexpect from 'nexpect';
-import { getCLIPath } from '../../utils';
+import { getCLIPath } from '../util';
```

Comment:
This method should be moved to amplify-e2e-core as well

Reply:
moved

Reply:
I dont see the moved code. It still is referencing `../util`

Reply:
https://github.com/aws-amplify/amplify-cli/pull/3749/files#diff-36366797da092f7d5734749205990a8eR5